.rubocop.yml

```diff
@@ -1,5 +1,5 @@
 AllCops:
-  TargetRubyVersion: 2.4
+  TargetRubyVersion: 3.1
```

Comment:
But Gemspec states 3.0.2
I suggest we remove .rubocop.yml and Rake task

Reply:
Fixed in d068fe8